code/modules/admin/verbs/mapping.dm

```diff
@@ -90,7 +88,7 @@ var/intercom_range_display_status = 0
 			if(!(locate(/obj/structure/grille,T)))
 				var/window_check = 0
 				for(var/obj/structure/window/W in T)
-					if (W.dir == turn(C1.dir,180) || W.dir in list(5,6,9,10) )
+					if (W.dir == turn(C1.dir,180) || (W.dir in list(5,6,9,10)))
```

Comment:
change these into the direction defines `(NORTHEAST, SOUTHEAST, NORTHWEST, SOUTHWEST)`